internal/services/storage/storage_account_resource.go

```diff
@@ -786,6 +786,30 @@ func resourceStorageAccount() *pluginsdk.Resource {
 				ForceNew: true,
 			},
 
+			"sas_policy": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MinItems: 1,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"expiration_action": {
+							Type:     pluginsdk.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								// There is no definition of this enum in the Track1 SDK due to: https://github.com/Azure/azure-sdk-for-go/issues/14589
```

Comment:
that issue was closed as wont't fix - has it been added to the swagger or should we be opening an issue on the rest api specs?

Reply:
That is not a swagger issue, but a Track1 SDK issue. As they are not support track1 SDK, they closed that issue. This comment is to explain why there isn't an enum defined in the SDK, not meant to be a TODO. If there are more than one enum defined for that attribute in later API, then there will be enums defined in the SDK.

Reply:
so the swagger is correct?

also is there a reason for this to be required with a single value? we could make it optional with a default?

Reply:
Yes, swagger is correct. Good idea, will make the change then.

Reply:
perfect thanks for checking - this'll be fine and be fixed by the pandora SDK then